modules/aggregated-dex/src/mock.rs

```diff
@@ -122,340 +121,83 @@ impl module_dex::Config for Runtime {
-			Self::AccountId,
-			Self::BlockNumber,
-		>,
-	) -> Option<StableAssetPoolInfo<Self::AssetId, Self::Balance, Self::Balance, Self::AccountId, Self::BlockNumber>> {
-		unimplemented!()
-	}
-
-	fn get_redeem_proportion_amount(
-		_pool_info: &StableAssetPoolInfo<
-			Self::AssetId,
-			Self::Balance,
-			Self::Balance,
-			Self::AccountId,
-			Self::BlockNumber,
-		>,
-		_amount_bal: Self::Balance,
-	) -> Option<RedeemProportionResult<Self::Balance>> {
-		unimplemented!()
-	}
-
-	fn get_best_route(
-		input_asset: Self::AssetId,
-		output_asset: Self::AssetId,
-		input_amount: Self::Balance,
-	) -> Option<(StableAssetPoolId, PoolTokenIndex, PoolTokenIndex, Self::Balance)> {
-		TaigaConfig::get().and_then(|taiga_config| {
-			if input_asset == taiga_config.currency_id_0 && output_asset == taiga_config.currency_id_1 {
-				Some((
-					0,
-					0,
-					1,
-					taiga_config.exchange_rate_1_for_0.saturating_mul_int(input_amount),
-				))
-			} else if output_asset == taiga_config.currency_id_0 && input_asset == taiga_config.currency_id_1 {
-				Some((
-					0,
-					1,
-					0,
-					taiga_config
-						.exchange_rate_1_for_0
-						.reciprocal()
-						.unwrap()
-						.saturating_mul_int(input_amount),
-				))
-			} else {
-				None
-			}
-		})
-	}
-
-	fn get_swap_output_amount(
-		_pool_id: StableAssetPoolId,
-		input_index: PoolTokenIndex,
-		output_index: PoolTokenIndex,
-		dx_bal: Self::Balance,
-	) -> Option<SwapResult<Self::Balance>> {
-		TaigaConfig::get().and_then(|taiga_config| {
-			let input_to_output_rate = match (input_index, output_index) {
-				(0, 1) => taiga_config.exchange_rate_1_for_0,
-				(1, 0) => taiga_config.exchange_rate_1_for_0.reciprocal().unwrap(),
-				_ => return None,
-			};
-			let target_amount = input_to_output_rate.saturating_mul_int(dx_bal);
-
-			Some(SwapResult {
-				dx: dx_bal,
-				dy: target_amount,
-				..Default::default()
-			})
-		})
+	fn convert_balance_back(balance: Balance, asset_id: CurrencyId) -> Balance {
+		match asset_id {
+			LDOT => ExchangeRate::saturating_from_rational(10, 1)
+				.checked_mul_int(balance)
+				.unwrap_or(Bounded::max_value()),
+			_ => balance,
+		}
 	}
+}
 
-	fn get_swap_input_amount(
-		_pool_id: StableAssetPoolId,
-		input_index: PoolTokenIndex,
-		output_index: PoolTokenIndex,
-		dy_bal: Self::Balance,
-	) -> Option<SwapResult<Self::Balance>> {
-		TaigaConfig::get().and_then(|taiga_config| {
-			let output_to_input_rate = match (input_index, output_index) {
-				(0, 1) => taiga_config.exchange_rate_1_for_0.reciprocal().unwrap(),
-				(1, 0) => taiga_config.exchange_rate_1_for_0,
-				_ => return None,
-			};
-			let supply_amount = output_to_input_rate.saturating_mul_int(dy_bal);
-
-			Some(SwapResult {
-				dx: supply_amount,
-				dy: dy_bal,
-				..Default::default()
-			})
-		})
+pub struct IsLiquidToken;
+impl Contains<CurrencyId> for IsLiquidToken {
+	fn contains(currency_id: &CurrencyId) -> bool {
+		matches!(currency_id, CurrencyId::Token(TokenSymbol::LDOT))
```

Comment:
This can simplify: 
```rust
match_types! {
	pub type IsLiquidToken: impl Contains<CurrencyId> = {
		CurrencyId::Token(TokenSymbol::LDOT)
	};
}
```